backend/apps/spaces/views.py

```
@@ -30,6 +32,32 @@
+    match = _DATA_URL_RE.match(img)
+    if not match:
+        return JsonResponse({"detail": "El formato de la imagen no es válido."}, status=400)
+    mime = match.group("mime")
+    if mime not in ALLOWED_IMAGE_MIME_TYPES:
+        return JsonResponse(
+            {"detail": "Solo se permiten imágenes en formato JPEG, PNG, WebP o GIF."},
+            status=400,
+        )
+    try:
+        decoded = base64.b64decode(match.group("data"))
+    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)